### PR TITLE
Commit template: use either Fixes or Related

### PR DESCRIPTION
**Pull request by flo-renaud**
Update the commit template to be consistent with the
commit message requirements described at
https://www.freeipa.org/page/Contribute/Code#Commit_message_requirements

Signed-off-by: Florence Blanc-Renaud <flo@redhat.com>

### CODE REVIEW

File: .git-commit-template [hunk 1]
```diff
@@ -2,4 +2,6 @@ component: Subject
 
 Explanation
 
-https://pagure.io/freeipa/issue/XXXX
+Fixes: https://pagure.io/freeipa/issue/XXXX
+or
+Related: https://pagure.io/freeipa/issue/XXXX
```
Comment (mrizwan93) [inline .git-commit-template:7]: Hello @flo-renaud 
IMO, we should explain when to use fixes and related keyword.

**Comment by flo-renaud** [comment]
@mrizwan93 I updated the template with your suggestion, does it look ok now?

**Review by mrizwan93 — APPROVED** [review]
LGTM

**Comment by abbra** [comment]
master:

* 043b118d8b9e61b12209c9e434c9c3c5d902da76 Commit template: use either Fixes or Related

**Pull request closed without merging**